CHANGES_NEXT_RELEASE

```diff
@@ -14,3 +14,5 @@
 - Fix: correct example for configuration provisioning (#113).
 - Fix: correct management of active attributes for configuration provisoning (#113).
 - Fix: first observable value is processed and forwarded to the CB (#73).
+- Fix: update registration requests coming from devices are not correctly hanlded (#109).
```

Comment:
I have expected to find in this PR only the line corresponding to NGSIv2 active attributes. Thus, why is this line about registrations not correctly handling here? In fact, it seems to have been added in another PR (see https://github.com/telefonicaid/lightweightm2m-iotagent/pull/110) and appears in L11 in this very same file.

Maybe it is due to a wrongly solved conflict when updating this branch with master?

Reply:
You are right, it seems that it did not solve correctly some conflict here.

Reply:
Fixed with 9fff0e461c6818687a51892cb62b6feae64e872b